binary/src/Pos/Binary/Class/Core.hs

```diff
@@ -26,8 +30,27 @@ module Pos.Binary.Class.Core
     -- * Utils
     , toCborError
     , cborError
+    , withWordSize
+
+    , Range(..)
+    , szEval
+    , Size
+    , Case(..)
+    , caseValue
+    , LengthOf(..)
+    , SizeOverride(..)
+    , isTodo
+    , szCases
+    , szLazy
+    , szGreedy
+    , szForce
+    , szWithCtx
+    , szSimplify
+    , apMono
+    , szBounds
     ) where
 
+import           Formatting (bprint, build, shown, string, (%))
```

Comment:
style nit: this isn't the right place for this import, it should be below in the non-Prelude and non-local dependencies